src/schemas/search.ts

```diff
@@ -8,6 +8,840 @@ import { regex } from './utils';
+                optional: true
+              },
+              biasingProfile: {
+                type: 'string',
+                optional: true
+              },
+              language: {
+                type: 'string',
+                optional: true
+              },
+              customUrlParams: {
+                type: 'array',
+                optional: true,
+                items: {
+                  type: 'object',
+                  strict: true,
+                  properties: {
+                    key: {
+                      type: 'string',
+                      optional: true
+                    },
+                    value: {
+                      type: 'string',
+                      optional: true
+                    }
+                  }
+                }
+              },
+              query: {
+                type: 'string',
+                optional: true
+              },
+              refinementQuery: {
+                type: 'string',
+                optional: true
+              },
+              matchStrategyName: {
+                type: 'string',
+                optional: true
+              },
+              matchStrategy: {
+                type: 'object',
+                optional: true,
+                strict: true,
+                properties: {
+                  name: {
+                    type: 'string',
+                    optional: true
+                  },
+                  rules: {
+                    type: 'array',
+                    items: {
+                      type: 'object',
+                      optional: true,
+                      strict: true,
+                      properties: {
+                        terms: {
+                          type: 'integer',
+                          optional: true
+                        },
+                        termsGreaterThan: {
+                          type: 'integer',
+                          optional: true
+                        },
+                        mustMatch: {
+                          type: 'integer',
+                          optional: true
+                        },
+                        percentage: {
+                          type: 'boolean',
+                          optional: true
+                        }
+                      }
+                    }
+                  }
+                }
+              },
+              biasing: {
+                type: 'object',
+                optional: true,
+                strict: true,
+                properties: {
+                  bringToTop: {
+                    type: 'array',
+                    items: {
+                      type: 'string'
+                    },
+                    optional: true
+                  },
+                  augmentBiases: {
+                    type: 'boolean',
+                    optional: true
+                  },
+                  influence: {
+                    type: 'number',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/groupby/gb-tracker-client/pullRequest?prid=8314532)
```suggestion
                    type: "number",
```